From 845ebe75d29ee1578378075e15c60192130f9b53 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Mon, 26 Oct 2026 08:28:33 +0100
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh 3176EF7DB2367F1FCA4F306B1F9B0E909AF37285 from keyservers/WKD: same certificate(s), with self-signatures our copy was missing.

Assisted-by: Claude Opus 5.5
---
 3176EF7DB2367F1FCA4F306B1F9B0E909AF37285 | 629 ++++++++++++++++-------
 expat.spec                               |   3 +-
 2 files changed, 438 insertions(+), 194 deletions(-)

diff --git a/3176EF7DB2367F1FCA4F306B1F9B0E909AF37285 b/3176EF7DB2367F1FCA4F306B1F9B0E909AF37285
index 0328cdb..8c8ba25 100644
--- a/3176EF7DB2367F1FCA4F306B1F9B0E909AF37285
+++ b/3176EF7DB2367F1FCA4F306B1F9B0E909AF37285
@@ -1,6 +1,4 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
-Comment: 3176 EF7D B236 7F1F CA4F  306B 1F9B 0E90 9AF3 7285
-Comment: Sebastian Pipping <sping@gentoo.org>
 
 xsFNBFzUcE0BEACzkr4qR9zoM63YCJU/oQTJEtt7SR9Hcvntk351O5QQbNJS55Za
 h+XfiAl1j45yrxP+ve3xU64Cl/GctZMLgkx8Qd3JECZCUkm72cvlBF1bJ0hkvcJR
@@ -15,115 +13,241 @@ u2K+zdFqZskPTZo44qZDjLd7HpN5SigFMCCSk9LTWcwpa4eSFcezmfku+dB5T79Y
 9pXTXHvPiQsYMKcMzOnbdXKvlMxF+dN3BT+uhEF5tyYgqSDaF07EnIJzdwARAQAB
 zSRTZWJhc3RpYW4gUGlwcGluZyA8c3BpbmdAZ2VudG9vLm9yZz7CwZQEEwEIAD4C
 GwMFCwkIBwIGFQoJCAsCBBYCAwECHgECF4AWIQQxdu99sjZ/H8pPMGsfmw6QmvNy
-hQUCZZgh3wUJDIYYkgAKCRAfmw6QmvNyhfRcEACU3hFFTVRyZTB+/Phui6bFhhbH
-oRVMZl7llwGdtmUR76moGAnOilKK3UG4Xn+yHk0Au0kMDciDzET1KL5pTk3FYaX5
-SdhMK5P3CQIRvGVQGEyzm2riGMGBQwbMTN1cWSrW39lORPDanDKIzzu0mvAe9Ufs
-M5Ecrz6xTIxMcMfBEaYH0snwMGFLowcDETk8DIM8qn6oOrH86S2+HP3LKeCM1DOI
-uAILj438lTgaHKfOmtAMxqwXzVtknQN5upIBvfDtYXeLH/zSztt3XIcDYrBVCFd+
-7wxvelu0C6e1yG3vQ6eQt4OAeSNBOXUAcIWsCti9uGL2//pE9gQs4s1ijJYFQuuE
-er3sTTqg4JU5y9NkDo6p9roZt+uDFSyj3wgOinfxMipNLniJpjrvV+tmqGhYZY0R
-WEP757A1M/xVaf89d9rp5pJ9QawNUIDfM8gH+m0FuX5YKlSvFak+uB9/Oeu+BKy+
-wWyBiEM3fOjnFBpAGz1nKGQFYvUuRtqFAmlLUhN3EA2ixL6tMvlkWmHS0o0o+YGX
-tANGcsS3KwWILlRarfhkHuc0s+gFiTKvfS/pTbiy6XbtYTtQ0n1HkLz32zwdnFig
-/do+xYVyb9w1IZSc3HZAA2h8NlW2crMbzHr4FlSF5p/Zk5gVdfnhuKqWsE/nKAjg
-/GwACVGzFbZiD2CqsMLBlAQTAQgAPgIbAwULCQgHAgYVCgkICwIEFgIDAQIeAQIX
-gBYhBDF2732yNn8fyk8wax+bDpCa83KFBQJjxyuuBQkI0+7hAAoJEB+bDpCa83KF
-cesQAK0yPR7YSvy+uumbMJ02Be/7bu60yUk5O8nngpmebzQLEDAOyp1/HTcNk8VO
-zyKsQfWOsCvupAvPpdlaTfXanJQa0kOBjos1B/hoc4lQg3UwpSxUbmAml8MZfWgs
-QOexENXh3kGQBHTQS7fObCx8P96BLgOvCzq/wo7dUf4cugfg0RQSVI6yQNUIesRq
-NSDHTRAKy6hWieW3itZRs/DCD9e/aesDAIGaFlxOWb5wl0mHHbYw2IhgK/RbSY7K
-kt24SPEom7rc7dD/ToG2qNv/5uUujSQdjQu0WE+5JVVOaYsKWkWTcdKVURhhRJQt
-FmBA+CQyC/gUmrPvjfWFk9LpbFi/5cFWaqWQpMjz0pQPyKcLRNNotYMaPWpxqIpt
-0sBtVBZSVHv1emyMkYccxgP49lfHpzWIdILZwKJyJ6PPtojV3lrcXc53ILsOTGi7
-iSSQDEmxwJ6hT0lzrJ0bGnTidO4pN6VqtpQUt3HsODtivodxfkGskuSMkPVunZLG
-4OIsdco/mdKpEEc6g2+dQnYl2tYFZ8w/l0gEakLbkFVIvsYdzCJpR39OJgRWE+YD
-aWWitak50L30tu5gXcIMw4+79s0gqUBBxYRo79bx31uV1fPcj+ajovLQD9o5lwfY
-TdPIaNPmQh6oyz9CVzTovUgsHP8Ji0Yepma09p89Ov1NFPlawsGUBBMBCAA+AhsD
+hQUCaVGFrwUJDl5I4gAKCRAfmw6QmvNyhUG/D/9sM/ikyxHOaPXDyRacnbJIkVLs
+CC7SAjj7gngRQ/zefSzhgxo12jlAd5KQxVuuNw7Q0agaldFSyxZgXNfS8qDv9uQV
++I9tovTgHMznW2tkVXqZTnJO5whL/+nLgZjS6K+DJk2BPfV543kjKQM/4m3dJRMc
+O5Rqbuw0naja5raxdeNsqe11rX9pX81S6g3fmfvwV+Idjf/bOvSNyjEMrrb4h1ya
+uZOOxj92jyXjrrXddRtSDtlBw4h04mRQIE8lvWeQH03GtgW8iVOU7BfIllVhQOeQ
+f522LPmA4M4ItRf95HltPLJuX32BEAiA+sxDNIcnnTsYn/AbeY0+zFrg5NOGbhp+
+ywjjalAQffpTMKwrNHNWupaGKtrxSBRequ8KIvYeCRvIgaUH+CyuNjWknszkCRpl
+takjuYV8t1ritBXU+eQTJVqBxCq6gECfbyvcllPX88pxxqvdqV6cw2rQT3ns/mgU
+Zx7H5pXVVGLeQxgajGi6XenjkRxjTLV37x3iprH+GKjdvGkvnKAJwXiR6JYPy7zL
+EhsGB1wCz0dbOtwdJNxQlFHQxM80Aiv0/Y6/3f91qrIW7lUG414MLWBgCIh+PQpT
+448O4viamdEO3Wyjhp05alWxk9JEEmtV6ZuKrg7jcyvplghID7T2DUQTsQjnWDBo
+aZKXft0EfDK0vsfqwsLBlAQTAQgAPgIbAwULCQgHAgYVCgkICwIEFgIDAQIeAQIX
+gBYhBDF2732yNn8fyk8wax+bDpCa83KFBQJlmCHfBQkMhhiSAAoJEB+bDpCa83KF
+9FwQAJTeEUVNVHJlMH78+G6LpsWGFsehFUxmXuWXAZ22ZRHvqagYCc6KUordQbhe
+f7IeTQC7SQwNyIPMRPUovmlOTcVhpflJ2Ewrk/cJAhG8ZVAYTLObauIYwYFDBsxM
+3VxZKtbf2U5E8NqcMojPO7Sa8B71R+wzkRyvPrFMjExwx8ERpgfSyfAwYUujBwMR
+OTwMgzyqfqg6sfzpLb4c/csp4IzUM4i4AguPjfyVOBocp86a0AzGrBfNW2SdA3m6
+kgG98O1hd4sf/NLO23dchwNisFUIV37vDG96W7QLp7XIbe9Dp5C3g4B5I0E5dQBw
+hawK2L24Yvb/+kT2BCzizWKMlgVC64R6vexNOqDglTnL02QOjqn2uhm364MVLKPf
+CA6Kd/EyKk0ueImmOu9X62aoaFhljRFYQ/vnsDUz/FVp/z132unmkn1BrA1QgN8z
+yAf6bQW5flgqVK8VqT64H385674ErL7BbIGIQzd86OcUGkAbPWcoZAVi9S5G2oUC
+aUtSE3cQDaLEvq0y+WRaYdLSjSj5gZe0A0ZyxLcrBYguVFqt+GQe5zSz6AWJMq99
+L+lNuLLpdu1hO1DSfUeQvPfbPB2cWKD92j7FhXJv3DUhlJzcdkADaHw2VbZysxvM
+evgWVIXmn9mTmBV1+eG4qpawT+coCOD8bAAJUbMVtmIPYKqwwsGUBBMBCAA+AhsD
 BQsJCAcCBhUKCQgLAgQWAgMBAh4BAheAFiEEMXbvfbI2fx/KTzBrH5sOkJrzcoUF
-AmHSMvMFCQb/66YACgkQH5sOkJrzcoVQXQ//bVDypVzLbR1pbJKy0ZrxxHzJQVBI
-6Ji2f4NsByIGV75Eci+cFyaR3JGZE021CHCDpzfBevxffz+Oyikftzb2/2Qq8GgF
-MTj3lLzkgq5py7H/498vyK1JmwXroQ8RX0X3iwAZncT5U5QI+GYZA9SkJe3ETFIn
-fkqZUdYXR0ZH3kt5ci+PxnwmD+HXAJUx/MWAX74Hi2/i+fkirpQ2tE0Kbo484Biy
-WTtfebajNLv97Sw/8TpGKgcLJKgBL0aY5QQoJ8dE8YQn8LNRDFk79YyZRHrXGpOs
-TpzEsQZD2ZS8YC3LmyWKhm/1lzYbKs1mlVbkodU6kkaJn8p19s5bI79gajjnoMm3
-Yh7q/Fp3nC2HpIpxWKnKqMdok/u+McivV7ue4VfRrKV7mXJ/0XUtcc8KYehDlDpn
-nbqETUkQPU/DGpUIxB3BR3ihEZJ6EpIkysXfgfpBPNvPpjnaV7+uPFgAzlst44FL
-RqQH0gI7IsBLABrgX+yioF1zQzOkp8iI8PJkuBmxeA1Q/M68VIIF4Wma3ej677M2
-ATdwX5cFqgrhvbwiBSst67YJjbyehwnl/tRAexV6/lIk4NnTKHPE2domgeAR+uLk
-mMO/o/G5Gk1cYbRXG38RM3vDqAAFEznec1pLla9UV6LrugJCHErnxI9Pm9h0njwG
-NJWpgz1NDMLlyvLCwZQEEwEIAD4WIQQxdu99sjZ/H8pPMGsfmw6QmvNyhQUCXNRw
-TQIbAwUJAeEzgAULCQgHAgYVCgkICwIEFgIDAQIeAQIXgAAKCRAfmw6QmvNyhVgV
-EACqcR2lm9HDyjUy+ChqGicRBDQqg0rZDohBPWICUWDjeGx/ZDouPU3yBoKWu3Lk
-0ouIPF0VshDOfab8H4UlV+06WcIEsYhV7zrUUPszCDrVsZt+2XSKsSBpdPaoBIVr
-0UxbN8ls/sS6TBNaN/1wdmTOqUXr62lwoF/cu+K4BzaRT5yaKs0SYb0mcqg4MSmt
-RPAWjlwZBvHwFZYyhaQap060uV5OGizfwZ3QLFldMtsSM5juQAok4Wod1OCOHYY+
-RahRaj/Z5bPgYE3ycM84G2j9N7nJu6MJLyyt2mOW49jjkgeKJ7aPledvuuyt3EJ0
-GQiJSQgjLXG/6wjmN+XmBJkcydalpMRxk/Yt6N7VHfet46vP8mC/jNbtUb5wFSGr
-RJcyZP0elzvO6VhSh1mlRoqEUHKkb1CFVCYGorfGdUuiRhTXKwPcjIqe1ZJQaIKy
-KOcy2CoItLNkcCNHtTloN5378Y6hO+IP3BIyRtlBcD3v9RIYsQj2Tu9Vnluk/Awb
-wLkmOnC6wAsDmmmhK6y0AVn3FpAZ7m17DpFiIlNpumNkjaULfEKFL2PPHB8VbYi1
-V1IAG69X+91JKxzTU83s+B2Rz8hei2jNbtYbj/E/VsiatOucOf9Yj8g0tnS4Xfjh
-a2VPXQQShBT2uhm+hF6esBappDgjoh9/AGEKD4Jhx47vLs7BTQRc1HJNARAA4MJw
-SQR1gZysB4RCS2HouDzCJkSUo6ImGYtRJogk97vfqhFvyDsvlST8n9jsizPUaC8z
-KwkZmIuNXhWa8lrHp6cx0FT9vCmzBz/pnSTk8r1/S7fDocWv5vmRtL44VywwbERZ
-oRBQ1deALPDXCNRor56cjcUxQw1m5sQzMO800pIx/Hangf2fx011rtKXTq4V2/OY
-mTLDmQ1M5mysAUEwDl+8z7ZouM4Xj+PVVxxp93zYL1G+kLs0kDbQCkuwtqj5SDLo
-P72Ml0TTOHZwvWhmHY8F7hMjWCzdd9ZahuqvfMcT0bG3Z8w6k4DwmapNnWatN9wH
-ds4TW4cp8nMVw6cULEZ+D0HygYKa/iHTRD+FHr2jrnmlS0sLb3L13KjGLa80mCHD
-btLZ2ww2UcYYCjPt05KWXNdO6qMKuvkkj81m4OoVGO78SswwIL2IZ+Top+fbAEX6
-NCpEgDNR/P6M2MXTRskqiEBXsHI4yw6UJyNsf2QFPJ5NIZbRM4/ippCXSSSN23nH
-p7Zq3Tg8kiztH4nawps4YrpPBy6HQ0PWqWPueH1sAw1MLN98hOA7V+WNv9Z1L8Az
-aYpenUZJwFRnXNzryHvlgJ7BkfUjEsuB67j/2H/jmUghbTkpaHuDLT/qncn8dLw4
-P5i332cRPK/s0ngeSAKdAUm8FIXW3lhnflKVggsAEQEAAcLDsgQYAQgAJgIbAhYh
-BDF2732yNn8fyk8wax+bDpCa83KFBQJlmCIKBQkMhha9AkAJEB+bDpCa83KFwXQg
-BBkBCAAdFiEEy43nCpDPv2w79cxWliYqz/vTrsYFAlzUck0ACgkQliYqz/vTrsba
-zxAAg/8wf9fNu6HPu477oUU1LuSTTTNAiYLVjTJjJ7LLZRaxIPgdR0TTkKgkIxLA
-delM3MOz9ppyROwqK8Qg1g+9PBsQY8w+q/0xUPvxPjqLBK4D5CDfYIMsCO5tvLxD
-nkNdhL+rjQJlovODs5BPwgfTd7DHOu2l0uudedg3WS3AEWiS/lmzEHkGEQl9sQcS
-p2g2TZJNC/mn8iStzkL+F58dIpdXhwlFig+St2evRkfJ68HmIa5H3MzGaQsP43Ii
-qpWrGNZA2MugR1j7Eqd7MeIhm8ZvqcfnkOixljn8geAtQmwsVxg18sVJV0DMXSnL
-4DmdjQ8lquVmm06Lba1KxJdfkQpYctNlRRyzW4GubJ1UUyxPdkL7ykfEvIoaf5/7
-z2N1QLOJ1D7njk5ic1PPFOMbK0b1j8oRUNhRYCb05JIsmMOm3hENUXrK17xwcl90
-L1ilrN63ZnYYy0ZrqDp2/s2EMX4kISGcyto9osdtacAE/jsAlnVxG9h8TaXIcHN8
-pbJYnnoFYf1J/unoPB6GogCjR6s8BycENxut+sdpKcUu9b1SkgYAa8ExdmoChk02
-xeqrs4wdG32tZlc4aR4YAg2fCVAhPd98oyi6dRL3HMszSRl1/sFLsrn7mByMk+HV
-vXK8FgCmKgEBdudpOaq93CvTcUuegaLn/TrvW3SJQZKpsXidTQ/+Ns4PriL1io26
-cpZKu7CMplwkJn5xIyoV+tFJHbrgmZ7csdo1QgLHyI0LCA/vIw22XCrT09it8anQ
-Ykbz/3JHQAnMtsJOZI8j6j/iITomNVcZZL334YxD0NB+aFJEZivIa9OCvFU5gQCg
-Ibzio1PP6Dt2PozKR63v/6hv0kp3SVlknWLRTkK3CwbtYhPmfkrN9HJVzyz+6JbU
-mTPk28AannszumPSA7fIMeIKmpgkTeDsozN6tlcqBDzRks0kQlHMzhGk5upCfgWf
-7aiqTF+clVSMyTiXLW+3xjN+AQHn4gHRB8buAoWaPV+J5vZF8xU6VL6y3fewfARk
-YLmZNy5Sp30ataBu5LPVPP61PMb0gskhAD/tYEC1oTuCSX7eWSYEA/GGjUkANXbR
-y1WL8YuDuckK5U9FHupkb3tZFtfuw/Zpz5rf+RzjBAPrAtCDkUCQD5Wg2F/KfVPr
-RxKCJ1va+21C6ngwmQKghDiIcjMUps9vnfixBOhTU2hg+woYnKnOOkg+0TD7zJPO
-0gV9s39W5B/2fdDfIprnq6dk5tJXcqIVUtnxlqUam1cMBKkgqSXQxLwvMUOjFBH6
-MxXmJwW3OVBZN+GYXFsVnDtCcHNqbEGEiQMIh+hhBbrStc+PiRCaudkiZ11ltx11
-njEdV7VbbGxepus3mTQuxVxU1GFdS3jCw7IEGAEIACYCGwIWIQQxdu99sjZ/H8pP
-MGsfmw6QmvNyhQUCY8crlAUJCNPswwJACRAfmw6QmvNyhcF0IAQZAQgAHRYhBMuN
-5wqQz79sO/XMVpYmKs/7067GBQJc1HJNAAoJEJYmKs/7067G2s8QAIP/MH/Xzbuh
-z7uO+6FFNS7kk00zQImC1Y0yYyeyy2UWsSD4HUdE05CoJCMSwHXpTNzDs/aackTs
-KivEINYPvTwbEGPMPqv9MVD78T46iwSuA+Qg32CDLAjubby8Q55DXYS/q40CZaLz
-g7OQT8IH03ewxzrtpdLrnXnYN1ktwBFokv5ZsxB5BhEJfbEHEqdoNk2STQv5p/Ik
-rc5C/hefHSKXV4cJRYoPkrdnr0ZHyevB5iGuR9zMxmkLD+NyIqqVqxjWQNjLoEdY
-+xKnezHiIZvGb6nH55DosZY5/IHgLUJsLFcYNfLFSVdAzF0py+A5nY0PJarlZptO
-i22tSsSXX5EKWHLTZUUcs1uBrmydVFMsT3ZC+8pHxLyKGn+f+89jdUCzidQ+545O
-YnNTzxTjGytG9Y/KEVDYUWAm9OSSLJjDpt4RDVF6yte8cHJfdC9Ypazet2Z2GMtG
-a6g6dv7NhDF+JCEhnMraPaLHbWnABP47AJZ1cRvYfE2lyHBzfKWyWJ56BWH9Sf7p
-6DwehqIAo0erPAcnBDcbrfrHaSnFLvW9UpIGAGvBMXZqAoZNNsXqq7OMHRt9rWZX
-OGkeGAINnwlQIT3ffKMounUS9xzLM0kZdf7BS7K5+5gcjJPh1b1yvBYApioBAXbn
-aTmqvdwr03FLnoGi5/0671t0iUGSqbF4gjYP/jOOTWQ16ImKxAbQGLoStrlU3ksD
-EvgE3Ot1BHkH+jLZWIs7f/OA9SM9j8aYqcHT+UCgiAiqDoU1axEf2AefpQi+5t5K
-llg42vHT3VILkWIby6LldYZzRcPKDdis2YX78WlRCpcN6poZTcAx5z6Msvj0M3Ud
-cgjshBgqtaa2pfwpGp+pmUh3tlz4Kqp+wTrooUsytuBLE/pvSE4cgt83j1Tu0bUw
-QQpfaFrfqGWbe9RGbSRNqLxPqKyackSumGvzmOjXUrzjb25/RAbWZzC1xVQT0/E7
-HbQSNtdi8aE9NxgsEVfH9iXHuxOOkKJMqHkrmFoyrFShYsjD8OmHCes3WsEVFgtu
-6ecUPvAcrDn3STC5Q71Gkoe7slbT8vBEJoENARCo1bKl8zzgIiNiuKQ4q7D/coCT
-BEbw45QFVJV+YoDqWUbpibbHC4Ox9heHo1qkjnXHjEsTNyUp9VN2r4aKVXrWdmb2
-ec+LF0Lt8SZScoLcr/OP4bXncHcokqTh4304A7myJXkSZqiVjssdkkts4KPKkMAo
-2Wx30ZjaBDTo4oRm2iK3rXFI6LS4ClUBG10JlwPo98Dkk9yzQxCvIMIGAS/PTCm1
-osxy7kZWDpiJb4wGBq0BZiCy23k/7TkeqkC/51z7U4mGZARfQI95wwwHxLURgJzJ
-suh8G7gkTcHL01kkwsOyBBgBCAAmAhsCFiEEMXbvfbI2fx/KTzBrH5sOkJrzcoUF
-AmHSMy4FCQb/6eECQAkQH5sOkJrzcoXBdCAEGQEIAB0WIQTLjecKkM+/bDv1zFaW
+AmPHK64FCQjT7uEACgkQH5sOkJrzcoVx6xAArTI9HthK/L666ZswnTYF7/tu7rTJ
+STk7yeeCmZ5vNAsQMA7KnX8dNw2TxU7PIqxB9Y6wK+6kC8+l2VpN9dqclBrSQ4GO
+izUH+GhziVCDdTClLFRuYCaXwxl9aCxA57EQ1eHeQZAEdNBLt85sLHw/3oEuA68L
+Or/Cjt1R/hy6B+DRFBJUjrJA1Qh6xGo1IMdNEArLqFaJ5beK1lGz8MIP179p6wMA
+gZoWXE5ZvnCXSYcdtjDYiGAr9FtJjsqS3bhI8Sibutzt0P9Ogbao2//m5S6NJB2N
+C7RYT7klVU5piwpaRZNx0pVRGGFElC0WYED4JDIL+BSas++N9YWT0ulsWL/lwVZq
+pZCkyPPSlA/IpwtE02i1gxo9anGoim3SwG1UFlJUe/V6bIyRhxzGA/j2V8enNYh0
+gtnAonIno8+2iNXeWtxdzncguw5MaLuJJJAMSbHAnqFPSXOsnRsadOJ07ik3pWq2
+lBS3cew4O2K+h3F+QayS5IyQ9W6dksbg4ix1yj+Z0qkQRzqDb51CdiXa1gVnzD+X
+SARqQtuQVUi+xh3MImlHf04mBFYT5gNpZaK1qTnQvfS27mBdwgzDj7v2zSCpQEHF
+hGjv1vHfW5XV89yP5qOi8tAP2jmXB9hN08ho0+ZCHqjLP0JXNOi9SCwc/wmLRh6m
+ZrT2nz06/U0U+VrCwZQEEwEIAD4CGwMFCwkIBwIGFQoJCAsCBBYCAwECHgECF4AW
+IQQxdu99sjZ/H8pPMGsfmw6QmvNyhQUCYdIy8wUJBv/rpgAKCRAfmw6QmvNyhVBd
+D/9tUPKlXMttHWlskrLRmvHEfMlBUEjomLZ/g2wHIgZXvkRyL5wXJpHckZkTTbUI
+cIOnN8F6/F9/P47KKR+3Nvb/ZCrwaAUxOPeUvOSCrmnLsf/j3y/IrUmbBeuhDxFf
+RfeLABmdxPlTlAj4ZhkD1KQl7cRMUid+SplR1hdHRkfeS3lyL4/GfCYP4dcAlTH8
+xYBfvgeLb+L5+SKulDa0TQpujjzgGLJZO195tqM0u/3tLD/xOkYqBwskqAEvRpjl
+BCgnx0TxhCfws1EMWTv1jJlEetcak6xOnMSxBkPZlLxgLcubJYqGb/WXNhsqzWaV
+VuSh1TqSRomfynX2zlsjv2BqOOegybdiHur8WnecLYekinFYqcqox2iT+74xyK9X
+u57hV9GspXuZcn/RdS1xzwph6EOUOmeduoRNSRA9T8MalQjEHcFHeKERknoSkiTK
+xd+B+kE828+mOdpXv648WADOWy3jgUtGpAfSAjsiwEsAGuBf7KKgXXNDM6SnyIjw
+8mS4GbF4DVD8zrxUggXhaZrd6PrvszYBN3BflwWqCuG9vCIFKy3rtgmNvJ6HCeX+
+1EB7FXr+UiTg2dMoc8TZ2iaB4BH64uSYw7+j8bkaTVxhtFcbfxEze8OoAAUTOd5z
+WkuVr1RXouu6AkIcSufEj0+b2HSePAY0lamDPU0MwuXK8sLBlAQTAQgAPgIbAwUL
+CQgHAgYVCgkICwIEFgIDAQIeAQIXgBYhBDF2732yNn8fyk8wax+bDpCa83KFBQJg
+izOmBQkFl/bZAAoJEB+bDpCa83KFOMUP/ReMxra8R4/G8kbm6miQ/nLNVCgyoCLW
+evK9dOILzLMlZcPz8T0L78r0Cce/Vy+neSiNCZuqc//UHaj8hjMlUl0tlH0XvJWY
+HYfpn8LXdfbu7zXUO4RYS3gphBJ025M72pl20vbFLdZM2ILHdO703awTmNdH7okp
+m9TX6ALIz4apaEVYhGmFVSeCh5MVzLX/YL0Ll3t+DcjNxD1XF2KeF5gwtw+Dia6a
+MxExXsypFlQLM9iRe6WblgFgNUBMd9FYGZH4w/7SOtyJUOQRg40rN5VG/1BUKimk
+nkPbq38LDoI4kkAoN+xSeCRYU4gjPuu9J3F7YpxXFZBTsI4HY9/uh5XVhrLuidKt
+huXp1O9DIFcGYIPvvDWOXwPKuDAskjCbsfHv9PlK1I7RbYH6ulkOlx42i7tNQOiT
+dlqAhsqGyI64sSh+RX4yDgogF2+ef9A6oB567s5YjLJdQVZtMdioPO5w+jL3zNQ/
+4nveab3QHr+r0cWXt3Z4I5e5y/JRSvDVCHsrnrZh7W/Sr10TJoN5wz/jN6KhNKHM
+x+q09Re1L6KdSISOKzxi2fBOYhYploMrh4ac70id1duR6jHX3Pzq096m78m+cBRc
+iGDNy3gggcYqjVEbnkcgAikytFo0wbolJxY6TMNT2yWA5kVJudOxJMfOsmWTGfqR
+qB1MLau9cqsjwsGUBBMBCAA+AhsDBQsJCAcCBhUKCQgLAgQWAgMBAh4BAheAFiEE
+MXbvfbI2fx/KTzBrH5sOkJrzcoUFAl6yxZwFCQO/iM8ACgkQH5sOkJrzcoU6ag//
+TGp12gxPSfLEf+J4W5YlcFggkDhYBXdXzlgzgBOOz7LEZIFOJqWBXjlem/5tWhfH
+LPRv19VecGfXQyaRm4tfu61DINECdaZBzHcFaAqdiV7pF+TtuDtZkYlkY4mLpo6H
++dcHiQggJMB5bBGii+3b5feV4ioD8kPvSLIH4JbCfu9/PsW0MF3xV8dUhiqZoVj/
+qqEPi/ZvDtLSZKs5egEX3VBQCkrmUKOVO0i3ch+1pPr4hz0jEl6HfRZdCpvELGHD
+QtAP/9ckR/SvPUvQrXfra6bKLkGOWbBkNIMmWSpzwfX7exl/mh8FASUyhJUM/S+9
+aBq+hSLBt+P6+WO4j1oR9pJITkr3X5g6ZEx7Zv/XxkKQJcgII9T8OpLsICS8WTB6
+IkEb4/R2AzNOClEpys5fQYhCZFUhjDjMGXDOibEG9pnh9LoZKCRfezRb/B1zu39X
+D06Qpk9ysizXzjlEf92y7N1ppVbDUEqzWeoBXwyhF8nZgskqSPGgGOhLeilDBNWP
+XZuFS4GJ4xf5KaoTXztvOZGamsNwfwTyZC/xJOKH7emPEP1Dw1W6kCMHfOUITJN4
+okhZi//aLFgvvPUJs35gB1FQQDRgK6lHhHu18V4den5u/2/5qT4c2SIaUX1TvZZz
+er7Luts4Op8T72liyB5zWZt0NUZ5p7OzVV+iw6kKIjjCwZQEEwEIAD4WIQQxdu99
+sjZ/H8pPMGsfmw6QmvNyhQUCXNRwTQIbAwUJAeEzgAULCQgHAgYVCgkICwIEFgID
+AQIeAQIXgAAKCRAfmw6QmvNyhVgVEACqcR2lm9HDyjUy+ChqGicRBDQqg0rZDohB
+PWICUWDjeGx/ZDouPU3yBoKWu3Lk0ouIPF0VshDOfab8H4UlV+06WcIEsYhV7zrU
+UPszCDrVsZt+2XSKsSBpdPaoBIVr0UxbN8ls/sS6TBNaN/1wdmTOqUXr62lwoF/c
+u+K4BzaRT5yaKs0SYb0mcqg4MSmtRPAWjlwZBvHwFZYyhaQap060uV5OGizfwZ3Q
+LFldMtsSM5juQAok4Wod1OCOHYY+RahRaj/Z5bPgYE3ycM84G2j9N7nJu6MJLyyt
+2mOW49jjkgeKJ7aPledvuuyt3EJ0GQiJSQgjLXG/6wjmN+XmBJkcydalpMRxk/Yt
+6N7VHfet46vP8mC/jNbtUb5wFSGrRJcyZP0elzvO6VhSh1mlRoqEUHKkb1CFVCYG
+orfGdUuiRhTXKwPcjIqe1ZJQaIKyKOcy2CoItLNkcCNHtTloN5378Y6hO+IP3BIy
+RtlBcD3v9RIYsQj2Tu9Vnluk/AwbwLkmOnC6wAsDmmmhK6y0AVn3FpAZ7m17DpFi
+IlNpumNkjaULfEKFL2PPHB8VbYi1V1IAG69X+91JKxzTU83s+B2Rz8hei2jNbtYb
+j/E/VsiatOucOf9Yj8g0tnS4Xfjha2VPXQQShBT2uhm+hF6esBappDgjoh9/AGEK
+D4Jhx47vLsLApQQQAQoATxYhBCwTgjuCNzEPohMDSTDRMv8P9Q7rBQJqVKpWBYMB
+4TOAKxpodHRwczovL3d3dy5nZW50b28ub3JnL2dsZXAvZ2xlcC0wMDc5Lmh0bWwA
+CgkQMNEy/w/1DuuO4Qf/TH/vbHGsW92SR23Smvp/jLLM5ANSvXvrLrJHjEYpyLKj
+YZhMa4ZXtxfJoLatzmMQzQ7+BPyfgD4ClB50tYhogoqhts5gaYrEv3P9nqVM/vQz
+vp1w0f6FVr0D0k2oud/ZAKSQJEfh/cMyWp+5x/ISaX5mG6t5S2aFhKoFg9YI0rKu
+j6L0U+wZ+8lTR36xVBWTBdm66AhayXSFdwTxdDCbG7bigm3OB3NrtBhSI7Is5CAV
+8JCitGoGat8oWjpUx5bJP1jquCOfoKyoO7BgMNQsPC7d0gLXIMSW0o9+rM22qVuK
+0KGrKkqsF1BU8Zqyrxc62SNHbez6KplfFywOf5/5B8KnBBAWCgBPFiEEq6Xk5/Tk
+B6vpyn7HQiyQZuIfcFoFAmpTLp8FgwHhM4ArGmh0dHBzOi8vd3d3LmdlbnRvby5v
+cmcvZ2xlcC9nbGVwLTAwNzkuaHRtbAAKCRBCLJBm4h9wWk6qAQCMu7aCEkHhj7w0
+kiNFSInsNrX3di6CFPyOJ6htonuXiAEAodSfXk8UjLVxxMcxwmf8sfkz6cCEERo0
+/PD1QETcXA3CpwQQFgoATxYhBKul5Of05Aer6cp+x0IskGbiH3BaBQJqS+5eBYMB
+4TOAKxpodHRwczovL3d3dy5nZW50b28ub3JnL2dsZXAvZ2xlcC0wMDc5Lmh0bWwA
+CgkQQiyQZuIfcFolJwD/bBH3FYzrmohKTehNlO43MOTEMPljpx6eauixHAORMEUA
+/iWhzxpeSpq9qaNUM7HdilIWh8T7czzRBXhxqJxFpPsNwqcEEBYKAE8WIQSrpeTn
+9OQHq+nKfsdCLJBm4h9wWgUCakBlRQWDAeEzgCsaaHR0cHM6Ly93d3cuZ2VudG9v
+Lm9yZy9nbGVwL2dsZXAtMDA3OS5odG1sAAoJEEIskGbiH3Bay2QA/jtFF1oHOSwV
+JiXX7yhpwQOC02OYKb4UYg9M3XGjSSh0AP9z2YSZXzo3Cp3aaOygHL+149Pla/TK
+3xlvNn3UydqkAMKnBBAWCgBPFiEEq6Xk5/TkB6vpyn7HQiyQZuIfcFoFAmnMDNwF
+gwHhM4ArGmh0dHBzOi8vd3d3LmdlbnRvby5vcmcvZ2xlcC9nbGVwLTAwNzkuaHRt
+bAAKCRBCLJBm4h9wWvkPAQC6/bNEvNY3CSMYhECBf78K7wBvwAEZzCJh9sb66Bo/
+WQD9G+YnYH2ETUDVGQnXfeE4d/mzAira8d3mUDb9zyhQmwjCpwQQFgoATxYhBKul
+5Of05Aer6cp+x0IskGbiH3BaBQJpwvyJBYMB4TOAKxpodHRwczovL3d3dy5nZW50
+b28ub3JnL2dsZXAvZ2xlcC0wMDc5Lmh0bWwACgkQQiyQZuIfcFrzsQD/QeGL/fiX
+2zTyUP8O1bFO6RGZmTty3AwoyIWlpmY5LG0BAMHQwPFKpNQOTAf/EaKQTOX6SqhM
+S5pnBOfAFET+/JIEwsClBBABCgBPFiEELBOCO4I3MQ+iEwNJMNEy/w/1DusFAml3
+/ToFgwHhM4ArGmh0dHBzOi8vd3d3LmdlbnRvby5vcmcvZ2xlcC9nbGVwLTAwNzku
+aHRtbAAKCRAw0TL/D/UO65wWB/9vWQy2eBqV9rHWItA0VV6ndJJxObcLmGx5XMhP
+LE/9/x9tVDIH77DJeen3yTzvg4I8KHOGqMyqIcKeRxh5ZofwKwWAObOf815wh1JX
+Itr8+/3w2xGGLt+Kkt0fGJASvD0YuAF8UGocO5ItLJPZJ1p8qAJBvbMs7US6epc4
+RRfHib+7dQYkBhIC9QneukteGczIxBp5PfWEQwJ4F1NEgVF0D/DqZhmYqk4s7vMU
+fMzTWap5Z9IbSieDte6Ai+8KIMQRqYC080cBiJtcrYQKrvuuDh9Q/iJETlinY6OE
+28XkiYD8FQHT8zeOtVI15eW9Nlkscjeb8PiFTKSXEtIuZmnbwsClBBABCgBPFiEE
+LBOCO4I3MQ+iEwNJMNEy/w/1DusFAmhH3x4FgwHhM4ArGmh0dHBzOi8vd3d3Lmdl
+bnRvby5vcmcvZ2xlcC9nbGVwLTAwNzkuaHRtbAAKCRAw0TL/D/UO674xCACwblyp
+2an8tk22DOnvARdgrsv/nHDTG15Ezr0KAzffyoGEfbDmhOaNAFO8mLtVEKcZzsJ+
+KnKjtHOKCrSpTaBSgioXPinWjb3poNogrnDtbjVl1JlJL2V5tEIuS6KXWjV9U+H9
+kUl+1Ltd6SEvSuiFl8s5T+T9BIwfT0pUGnY54p55lf0xbFu+jLtcDbtzU73tv7/L
+6PQmbERNebUzatYNvXp8dmrfc02ffcrFENzPXwIkiGeWtS8RLlj6dOkI7Ar86096
+GH5JW7POgeCoB4oVNXDv7VjXzs0SrVnBSlD1FSXpSlNBj8OU3r42woIAdD5ZEDlI
+tNmGx6nHe+gfEm1jwsClBBABCgBPFiEELBOCO4I3MQ+iEwNJMNEy/w/1DusFAmgf
+6bIFgwHhM4ArGmh0dHBzOi8vd3d3LmdlbnRvby5vcmcvZ2xlcC9nbGVwLTAwNzku
+aHRtbAAKCRAw0TL/D/UO652KCACirFpWU6XeROYyOPWb5ROoHP+MrtLZn5pRWWrm
+RecnAeQVYYZg5649PgN/chCsZlOSLEHawQvfQN8QWyS685gx5AEH4Mryg1AaCWXm
+FqJOeSuafTJCpzkNyaal1kaVK2thVPaCC24pMg/HN0/O/rE/Va9By5yNGlhqzdcT
+UZqDvcw/F/QCsygxFIGMjGTR0TvX0VchVpHoMSLIyr9iCW1W4uc1AAiGgE7KX1xq
+k74TxK9Ldc5e8Ir3u/Io9R+earkYbnHGaxEXlZeXCOoD8BJn4HZAtSVowwpiWUU2
+zIbpl36ESe3MrKcAl7QB6E+iXZ6gVEcGnGpbHa5fbviqr08LwsClBBABCgBPFiEE
+LBOCO4I3MQ+iEwNJMNEy/w/1DusFAmY+jAMFgwHhM4ArGmh0dHBzOi8vd3d3Lmdl
+bnRvby5vcmcvZ2xlcC9nbGVwLTAwNzkuaHRtbAAKCRAw0TL/D/UO66SyCACNKAri
+N/ihz1aGnFZNirxMzZogsQjCGleunIeJuFoqLPYrpc1pgecn5kQmx1ZPMXEtCJwo
+YnbAxW0IgmMInef/UwMyxWxQerKa+KxKEw5xYT78g5YYmGi/D923bABh/kKilPYV
+d1Q+vey2Z7QXjGq1HDhSjflUyObtQ1dW6eSJgSUP81O639tb0PLh4q/wu6V+mIlX
+KMWkW8TdM1ZOdXO/n0ZNEy5wQW8v0YjZae8cqTZNh9iP5/YPuirxL1m8y5GFFdDA
+Lz799rY4A/7t1uZI8DAXx1Him9Qgj7YO/uEnqdSoe6k0z0NFZYfq+6Gk4NRsUrD0
+ASiggA/LxjDc3exLwsClBBABCgBPFiEELBOCO4I3MQ+iEwNJMNEy/w/1DusFAmRd
+NDAFgwHhM4ArGmh0dHBzOi8vd3d3LmdlbnRvby5vcmcvZ2xlcC9nbGVwLTAwNzku
+aHRtbAAKCRAw0TL/D/UO63L9B/4sHQRIRN9OSdZU/GUkXopLJ1Znn6Kb5HXr53G0
+VhqBCMBWp+y1dSk76WIWZX2zj7CQy91QR3T0A6be6ifIrM2FP64lmzRRne5mOEWU
+T3+lwJtqaTibtrOcyPY9I9gMEktnRcAj/FbbyFTWS/kzXBT+3xj75JQFXu+nGPOR
+4HSevf0hQKYnooiQIAGP8KmvIOH7HUSBWnfLxUHn+B8juZaVHiqk2ogZMSr0ZMZC
+iyqbfE4ZCaWe2dVEHXjQsbQ5/wuh8ABHY4xBfX6FiebKEWrBc1D7AbN/ZOLcbN04
+ZyrQP97O8BnjW8zvtFfAXWMKyTgIQU7BTpZJsrcTyjT/FJcEwsClBBABCgBPFiEE
+LBOCO4I3MQ+iEwNJMNEy/w/1DusFAmJ6rzEFgwHhM4ArGmh0dHBzOi8vd3d3Lmdl
+bnRvby5vcmcvZ2xlcC9nbGVwLTAwNzkuaHRtbAAKCRAw0TL/D/UO60T5CADmIo88
+rk4fHIwVUEaXYQV30elZsRjtBLStJZi7v3Py4HinqkWQ3cGixQgS4RBbs1gsjmnC
+dFfTt6oi+0YJ4TbYl4YIsfQAjl7KIlxcDo6slG4u0u38Yzc5vSUvxmXXNVtde/fd
+3qRRUKI9y2ZgHVKewkAdceWFarg8SvxYux8beOhy8jAl5MXz0W3p69vqDJlKXT7C
+JtYljMMvSGtPIGVw0HGNOHrgJsnhOCheChKOqMaKeX1pzYeKmPvbVaXGfPJ7a3dW
+wzCJATDQfTVb1nIxrh9XNfl1oV5fmiWD0YmKGeJf3NrJfGjvAZXetyDepmAgQWxu
+Qis1J7DoJmSvRxmtwsClBBABCgBPFiEELBOCO4I3MQ+iEwNJMNEy/w/1DusFAmCZ
+e7AFgwHhM4ArGmh0dHBzOi8vd3d3LmdlbnRvby5vcmcvZ2xlcC9nbGVwLTAwNzku
+aHRtbAAKCRAw0TL/D/UO6+r2CADB1q1g9jSBvgtoOpRPx9Jr0A4JYSoFv5cXcOIZ
+VouxICwwczvUxJ2TG9ddVvlBXCcU9NEYLfQtezsfMSrMGoiLdC+nneDhf8Fw1isc
+8wK82EO0qf23bu6dw77NzGVj3THrHnyWLLFCNLZJuiXPBFhG2GKw3PEDjkSAlxTZ
+HeXWuCNzC39f9bcW4G3ftiQQUWArk6PdGIfY+M2zZ3Z/W9+3//ULGKDf+ws0iG+P
+/FV/J4eUCeiv5n9D6/DFxcJ2c5HS1LgMJs/j10PZc010Kkm67U2zBqxLzpyOOGu5
+yGD7bbDBAUWJYu+yJeN9cP2p0xjpsI0PJAXZ0yF41TghhDmxwsClBBABCgBPFiEE
+LBOCO4I3MQ+iEwNJMNEy/w/1DusFAl629s8FgwHhM4ArGmh0dHBzOi8vd3d3Lmdl
+bnRvby5vcmcvZ2xlcC9nbGVwLTAwNzkuaHRtbAAKCRAw0TL/D/UO68YgB/9kV1UO
+YkFrvukODCz1MMCDfvdDi3i3boOKan8YB6M/0vzkdEYTdwHCmnRTk3a6gTIFCatC
+eh76CBoQ5xBdu/qrYYh4DgtGkztjD9SZ9tx3spCFGy1mIldvZ5y2RKWIO94CetJ9
+LjIHqmOczrhdMJloowJc6nXiiJHp7ywv6nhAj3LwkVG/My2BAJuYVbkmlJ46Et2d
+uNvrKpZnzNATHeUv9N07OT3PhSwvamJ3K1xQx7NgfcjOolH7FMwy3cQtBZS0QiVq
+DLga9eQoxM4re2q13ec8oIF8AtgR+G66geiRMuShu8Axe2I0mBZQJt4k9pb5sxk7
+dPTttUJirCrsHcpkwsFzBBMBCAAdFiEEY+sE+qMMduKVLm7WVllTuVNydWwFAl4t
+/ucACgkQVllTuVNydWx0pBAAkVqOKSHfFJD+fj8tHPZ4ASIjhN4/GaQMrGaXUylm
+mTEjVKvIXQN3rvA/GJnjnPRhnYY78Vw/YFV6LkDx62BDT9oW3VZnnrlmCO1l/56e
+2toCpTsQTLoyE1Xnf+5EG0UZ8MdKYas//Ye08nNy+qac9W3JQEwoYME7EI0HeK89
+wOy7zR/5d36O6evASLWnunGWYI/RmYkWPRTkF/ZBGR9V9e8vpCoQA4FNSoJdN+ZO
+ZnVOl6cHya5YOCqiT/dCZ7xj7ShN7c7637QhHGLZXWUOQHUN3VA58RendZXgt7Vv
+o7+s6BPl9bRdQRAn0I3nzNUSo9PDpo9b+osDrlR9uSI78fngGc54dKUVT1g+sRwJ
+/aqihMepnJQSAR0HkDrU2Gtq87vz+7CBocaIgrOimMuvwPkt71hMPngMCVs/pf7Q
+lv3du5Vy+baub1wohs3qbBsfmCuLtRIMjRQBTJJVh5sHVR0WY45gui/MM3tSz+xY
+liSyZZenDvQhkuZJXERZH4i6tvM+CX63CSxm/kQ84+FVremXJFj4BFqSKxWa11Fb
+ME3Bt3pEwbW6tGJZfIPrSvlduJc3ew+9N29iSAUpYNDxZL/4DJ7DOedThlTZ4sVh
+FmLhNa6DNplXIC/sJEmLVQw5dJp+pb8yVxaUzYarDhaEQdfvRgpBEnALbIZYXY/R
+Pd/CwKUEEAEKAE8WIQQsE4I7gjcxD6ITA0kw0TL/D/UO6wUCXNXDMgWDAeEzgCsa
+aHR0cHM6Ly93d3cuZ2VudG9vLm9yZy9nbGVwL2dsZXAtMDA3OS5odG1sAAoJEDDR
+Mv8P9Q7rGBEH/2KBEdzNzi5FE+w4FomQWpDbN4n7+t37wQ1YcGXHq6G0sZaoxRUC
+uggvrxbppMVdIRSL0vcnwkyzWDHC2vAvU7liUqjrU1yxqGA2w+rRDkGzJyiflv78
+qW39ValR5abcJD6TXeUemnLBL+KzStW28w1Vs6wrywUbBx4sq/DBR1YyYz2RakH3
+Oeitghzb+2IZOodwxv9C0Jdi26ccXGCpBfoMOQMCWswX/w8KaJsMqzEUZoqnICdJ
+A8gzvZLTHfu8/NbddhurYhBVaIsjwpmJFNAo6QK4NJzAVjM6HK694u0TTueu+VcH
+nxefov4QcpMmhorp5qSeRteiWjwL7NXizonCXQQTEQIAHRYhBD1+lZ2J+s/uODcZ
+IbALxmpAGhYABQJc1HOXAAoJELALxmpAGhYAwxYAniPJOey52Zkpy2ULNZXpLnMf
+U4ccAKCRtqJckPvSBE69v8XIF9imvxX09cLAdgQwAQoAIBYhBCwTgjuCNzEPohMD
+STDRMv8P9Q7rBQJqVIBQAh0gAAoJEDDRMv8P9Q7rYaMIANCqXp6n/dq0CGNCHFsu
+Zw7PxoG7a42dvhtsWFHxVs+LyYT5QWa5FoRbDhfXqPwxQdqCOd6JXvlEQJW66JDs
+6Yw+TJnLQ2asE6RefO3gp463Nwi1MHsh3AnLOz+h/58shfikK+OYxpm0BDZWnJeu
+U6bC1qBVjIFN9g0gCpTFvysI1jFCdti+ns+WeXrch31lZ4yTIo3Sm2IkFqQ3WE60
+rvMNruonOUx5uALHAdiXisXozBq6sYwRalLM5nZRIjEBFQhS+JoxzrKObj0wu9W7
+xuxE/xqdCCS/k9vrpNG52ORQ7QBUcWVKr4tVzm29cxjbaQZMdE2WN+acK3zI90uW
+h0XCeAQwFgoAIBYhBKul5Of05Aer6cp+x0IskGbiH3BaBQJqTBj4Ah0gAAoJEEIs
+kGbiH3BaZrIBANCKGJ8ejXfJlLug/I436PSWi48+G9OvLhidlRVFTKdLAQCGPcl8
+h2EuV64RXLYQbQ+zd7x7oEdcIJS2Ha7DgujAD8J4BDAWCgAgFiEEq6Xk5/TkB6vp
+yn7HQiyQZuIfcFoFAmpAj7cCHSAACgkQQiyQZuIfcFpuKAEA3ANdn+JPgp3WYieU
+NvO00ellb+08puag2QaHX0o5oVEBAMawweIb4Y6u3eESD+nsWZ08dO+L6NiJsstQ
++cw6fIIIwngEMBYKACAWIQSrpeTn9OQHq+nKfsdCLJBm4h9wWgUCacw3UwIdIAAK
+CRBCLJBm4h9wWjxKAP0cCP4mHHtnm5ogl1/WZzWswNcRQ1vgNht1gciF7H263gD7
+BJHrEmZOa/jR2BcGcEZArAOA8S0IHLWzSQJXm3nMJAXCeAQwFgoAIBYhBKul5Of0
+5Aer6cp+x0IskGbiH3BaBQJpwybsAh0gAAoJEEIskGbiH3BaHx8A/i3sJC29uFt/
+LvH8wCYnp+NlgNXb+EnqPVfVNqmZ6wReAQDrDqAVBDh3U/qF5n/Qsxknv1N3nOxU
+LnuLsNZ0RYaTCMLAdgQwAQoAIBYhBCwTgjuCNzEPohMDSTDRMv8P9Q7rBQJpd9cy
+Ah0gAAoJEDDRMv8P9Q7ru3oH/j1XlO6sEdKFfEUWWEU1EvZOooaeNMWPlZ8+rl7p
+BLlkIKrq62KuaNi1G3b0iHQnlk+nEduXOfilBMPFaJvgGwGTGENDxlkP+7UM3xua
+PZplsxqrPB3al65X1bt0uFAd9/vgJz8Ve59cIi4qWGYwmhliRpokBvnaZCCKLH+p
+uk5c/4/PoCI36/eH/kXvFUpf7Zgp6iypU8G6D6wWExtzV44vjJ0HrDqRjUcmA6fk
+a4kPgoMnyvJedyitKmq8LWdPn6IXhiw63lmsPmbsmZH+PQNpXm7FGOXFP2VkiLZU
+CozQGgPKe3lPznkYKKY3l/gcEj7EkdZpdq98mAu53p/SZ1HCwHYEMAEKACAWIQQs
+E4I7gjcxD6ITA0kw0TL/D/UO6wUCaEd27AIdIAAKCRAw0TL/D/UO6ycXB/9jiE6Z
+gSoKy7RbbmHBLAZWUwRaqBNg+BMSGZC8EzdTY126qQey16sYkzYxZlXbWvuMXZgE
+/Xa7inzqMVrRyEOvFMFi1Cx8/2wfPWTSNZv73aAxpK7fAxFR52CVXuVUVeXVv4PB
+qI2cN5NlpE5CWOo0G4Gv2ID+E/naAp7TkKhMr79AR5mjth3vujpUz88ngRKA1cn0
+5bo1mSazEMs4FUvW68AgtGIY/V86oCS4vcBYePTZYM5SFW+IWJq9uIpcCBAdCDi7
+/Ku8+553TbavbzwEGQqUaRWA6SiLMLeHODyc4Qc27VtEtG+BUaZ1sMfCN0VVfeiF
+7KUIvHl/Z2twLqR/zsFNBFzUck0BEADgwnBJBHWBnKwHhEJLYei4PMImRJSjoiYZ
+i1EmiCT3u9+qEW/IOy+VJPyf2OyLM9RoLzMrCRmYi41eFZryWsenpzHQVP28KbMH
+P+mdJOTyvX9Lt8Ohxa/m+ZG0vjhXLDBsRFmhEFDV14As8NcI1GivnpyNxTFDDWbm
+xDMw7zTSkjH8dqeB/Z/HTXWu0pdOrhXb85iZMsOZDUzmbKwBQTAOX7zPtmi4zheP
+49VXHGn3fNgvUb6QuzSQNtAKS7C2qPlIMug/vYyXRNM4dnC9aGYdjwXuEyNYLN13
+1lqG6q98xxPRsbdnzDqTgPCZqk2dZq033Ad2zhNbhynycxXDpxQsRn4PQfKBgpr+
+IdNEP4UevaOueaVLSwtvcvXcqMYtrzSYIcNu0tnbDDZRxhgKM+3TkpZc107qowq6
++SSPzWbg6hUY7vxKzDAgvYhn5Oin59sARfo0KkSAM1H8/ozYxdNGySqIQFewcjjL
+DpQnI2x/ZAU8nk0hltEzj+KmkJdJJI3becentmrdODySLO0fidrCmzhiuk8HLodD
+Q9apY+54fWwDDUws33yE4DtX5Y2/1nUvwDNpil6dRknAVGdc3OvIe+WAnsGR9SMS
+y4HruP/Yf+OZSCFtOSloe4MtP+qdyfx0vDg/mLffZxE8r+zSeB5IAp0BSbwUhdbe
+WGd+UpWCCwARAQABwsOyBBgBCAAmAhsCFiEEMXbvfbI2fx/KTzBrH5sOkJrzcoUF
+AmlRhd0FCQ5eRw8CQAkQH5sOkJrzcoXBdCAEGQEIAB0WIQTLjecKkM+/bDv1zFaW
 JirP+9OuxgUCXNRyTQAKCRCWJirP+9OuxtrPEACD/zB/1827oc+7jvuhRTUu5JNN
 M0CJgtWNMmMnsstlFrEg+B1HRNOQqCQjEsB16Uzcw7P2mnJE7CorxCDWD708GxBj
 zD6r/TFQ+/E+OosErgPkIN9ggywI7m28vEOeQ12Ev6uNAmWi84OzkE/CB9N3sMc6
@@ -135,18 +259,18 @@ yhFQ2FFgJvTkkiyYw6beEQ1ResrXvHByX3QvWKWs3rdmdhjLRmuoOnb+zYQxfiQh
 IZzK2j2ix21pwAT+OwCWdXEb2HxNpchwc3ylslieegVh/Un+6eg8HoaiAKNHqzwH
 JwQ3G636x2kpxS71vVKSBgBrwTF2agKGTTbF6quzjB0bfa1mVzhpHhgCDZ8JUCE9
 33yjKLp1EvccyzNJGXX+wUuyufuYHIyT4dW9crwWAKYqAQF252k5qr3cK9NxS56B
-ouf9Ou9bdIlBkqmxeLF1D/9hJ0VQU3BjaBVJTKYsfZsvSDLxgqGHFdz5TzyLnGap
-shR9ZMvoyjUZUylupYpvnbKx6ypxZEP7ML+nWZ45KYdLSsaqXHN8mCInlD6MvcqX
-XccuUNffcgQXHhlP/lE2DFqlYDjhC13m5UyAqFfhk73kPQ5MaUX9lPZCfFYW+YVA
-OJlAtlns/fbnJY8EJe9msj+0aAyENdhq61+XxUIvtZp8KiT7vMtgFHgkguw888n4
-GQvYSQ/DvCnh9N1wnn6En8QDuBx/RK9VDfU0hUs7CH7SvxuHFvF6Koe0v+bS2kZO
-/3QhnLZhDNOkm2nMImRYGD+8QJGIIOebRl/FC8VUCOCqY1z90S2sOWt0VFJvUoO/
-m0IFNYwdKsati7l6TzFpmqYl+rlsfDya1idtVzZx5nLM8Erck1gmDGh9WXHasuW1
-nvaJJSClQbrPJNiLtEZN+IZj4Ggd0xv8Mjq2lJzWXsHHMRXPZpbOaNb+PJa5uGRE
-VvXY8FCI/3glOWd327m3ZN9keRPTbzJXMXBPuUhyhSXdB9nFWT54SORvdD/3kBQJ
-JhoaRraUkPQJh/az/ivxgdSY+BjNrtyA8DIHbCYUxlHyYdVFuqYQceseVjDf568p
-Rb+1O7KYIBX0rZStwJisus0d8ZboJppinOuVwk5fyhu8LpxuVbrRPQQsZb4lSnhO
-XsLDsgQYAQgAJhYhBDF2732yNn8fyk8wax+bDpCa83KFBQJc1HJNAhsCBQkB4TOA
+ouf9Ou9bdIlBkqmxeBc2EACM63yykzLfavYZmhj56N0+pxoZJjQwXcqzlSwakWWG
+U9VawxG78PvQVm+I1LqkMqh8jc348nHxAecm2j82tkiDsuug6l6sycTYxmZZhRDe
+SZwCoX3n2jbtJg1+3m0u13DK9IIjuCifsZYB1/X/YXWNj54ybOeDHwoMuI7JvvnC
+uQRHTc5m+rnsY0KXDBs0xabtLXstBwqmFxpxWcdMYBii4d4OBNzeAjOxphUgj2aG
++ThvO4g8O2jVRtKZkDepnzyayFKOntSCNIpJGO6tg+FNJUU1KbX/HGuvy1fldgWe
+Z5mkKjNXq+te7xJFXvPcQ+UIwvj5pnveGnjn0jV4uBBW3dRbl/lUhQUIy49Y+B/8
+Smkj92igyYDqJJj0Juq5oVQVoFqCKAFl0lvH31c4BHKiIBsl5ByirZxm1R5bbPiF
+dKwYTVADLmRil9pXUJ8iTQ3VKEfDzTvDQlji1TDKl2rQl5HTbLCT6vpEbRRddS9r
+XnitoGZ4ALxqv7LpPBW4M0WLwvq7a0qlBRXWJ2MxP/3S7IQqLn8GAewdanDj4D1t
+W2R827KxkjDEq/zkXqzUMyGtZTV9hrqmXwkiKOzJHa9J1TFVzjSaMwcgc27Vn1ep
+q9U/idG096sNWCa/pTXbo64TG7feuP58x7DwyentmZysWsjN7eRzlaWr8Xf5B/u3
+5sLDsgQYAQgAJgIbAhYhBDF2732yNn8fyk8wax+bDpCa83KFBQJlmCIKBQkMhha9
 AkAJEB+bDpCa83KFwXQgBBkBCAAdFiEEy43nCpDPv2w79cxWliYqz/vTrsYFAlzU
 ck0ACgkQliYqz/vTrsbazxAAg/8wf9fNu6HPu477oUU1LuSTTTNAiYLVjTJjJ7LL
 ZRaxIPgdR0TTkKgkIxLAdelM3MOz9ppyROwqK8Qg1g+9PBsQY8w+q/0xUPvxPjqL
@@ -159,75 +283,194 @@ mMOm3hENUXrK17xwcl90L1ilrN63ZnYYy0ZrqDp2/s2EMX4kISGcyto9osdtacAE
 /jsAlnVxG9h8TaXIcHN8pbJYnnoFYf1J/unoPB6GogCjR6s8BycENxut+sdpKcUu
 9b1SkgYAa8ExdmoChk02xeqrs4wdG32tZlc4aR4YAg2fCVAhPd98oyi6dRL3HMsz
 SRl1/sFLsrn7mByMk+HVvXK8FgCmKgEBdudpOaq93CvTcUuegaLn/TrvW3SJQZKp
-sXinJw/+PtR1IOXCq30k12TbLjlOwJ5cm1qVOyZfGHcX3rggnviOVt402KpY6QkR
-lgQB7RN7nQd+yVykp/Mh+9Cc10A1i/fhtcKpOYsJL7ZiqgGqcyRbmfZWDZTRmA8S
-PX6E6vfX/joXhZ0951JvCAPTdDMIxgiCeedNRzPjTvFspVaMtGPnq8W8kBgthMu2
-I1WUrgk2aNwbmIkgj/AZnfah8QiZgRkIG7QP9kAXC2fKP8/cNFN/SMLHovKGnGZ3
-FUyBZwQ7FNxNuTNXskPFXXAKiy+791BTovttMwQbPoPIwQXc11FYafkQLF5XaPIv
-6rXQ+P/dFF0+xQV5iJhrczlZs2tt7+w8quz/2bnhAKoa+77Vve1EO/mJFw2uKx4u
-B6xiQpa3cYsRMgkz6w/vkr8MQLnd4QFCmRfYTm9Hw/QO3J+txBEVYGiRgIu6fTqq
-m6p4vCTrLEBek3glBT/l950ePosvBF+LHHzei1Aev5p4aEVWdxjRMzwGYvTxnvEa
-lialDLawVVD1cXMAIH4oz5pjCFyFMmZOcrE0Hk93iVXd0d8sTkX2h0g+ZLh511sL
-Kf8mJbbCyP4QbVWvDe12Xc2fSQ0/HyFo+edeQ7H7p07ZLKzm6UXYquJcQKGKom6C
-Pi9QQh8L2k8vYIV4YaYJ8ptAyZNm5rnEWoq/emqU83WKLM3e4izOwU0EXNRwTQEQ
-AOhMwHA6FxDjdxLDnPYZZ/HRCB3j+Fn5s+c/qiK3J54G4yYP91871FjDeF7pDsmc
-QRgCz0k6GeZOzFOkpCTGg6aMPkOiBo931OqckzhlACnLSCzR5b2bILTaUGnf4t41
-D6+tCFK2dfJBdQ0yYfB3la8kg9a7vtnlaM9UO0Tr+o9NYOWysUAa5fxS9jSF2Czg
-eZ6k9Wa0bj90u8N9cfsGrMB7F6TVPG4Tf7GbCvgMwaBfSQK74hXVWd0wjTW0VGIp
-xRfAYudJyB/2da5rOsMWh5hEe6dShwEQ1tJHnjuBIJI1UZSyVtFqMj8NysftD7+V
-rd6N3Fp5umUzc6tViag+u6s8Q8TxCXMaSwoVtBV1HHbqKiCzwd4XNwHfv/h0VrgM
-0SXrYVmHwUkLUNdOlAKWRZ7ExaTMx0oNaKwjr3FhV7W5utf6kQ9lMfS8gV0dJM1n
-Zp0Zkgi/ojuIecqBQXJwTp1YQo1QmJHM0sKTu6pOOlTxizaT4Ak0etQf9SLinltM
-eYEdCoFavWkWXIIP4YM94fuD5Ekc03b2iiCMKVONSr4dKaAPFEtV3uFIoS/VwG5Q
-Q8mEhZZH9ymOeUrm+YvljFSfp1TDp9dGiYNKCx52Zj7wChqswzVEVFTqGEZqsYty
-uuDQM0JhX6TGT75zmsqiJhBGl6nigGrdaRCnWvWv0n0dABEBAAHCwXwEGAEIACYC
-GwwWIQQxdu99sjZ/H8pPMGsfmw6QmvNyhQUCZZgiCgUJDIYYvQAKCRAfmw6QmvNy
-heNyD/9uERCjRwJzBaF7eOl64Rqt27mYLgJKU9eA+4xit5ZEATPAlTuIbbqp0edY
-QLo2peMOgAADWmSYujOTUK7+e0/hHo7Mjr2RwbTnKOtKrG7tj4emqSQ4la0LGkJ4
-J1RoRHPLQw+VDOS7XHkidXwU4RJzz20nZGXVfCHmbvNOzoqbhUC36Sh6dONFcyUN
-Lk30crABh6vGVqKRL9KM2A3CH3RU9reLb4khKgTUlDWt0g9itU5V6rih2dyEkApD
-t0OOP3ksywTCqJNBSnVYopVn1IlkEGbCKEfeANJteDQpu3P5RkCu8pfvkYnOdik+
-FbmfmNCtfe67lmgT0uzdfMRzR9Pz+24Tkhruzxlt30m14n+8QHwfZqCAkedBiyG3
-vf4guOunrtn6RgeeXZHfKBgy/xhsgg/RJpDUkoGeGDSrdn0elRbSXc57WaytonIa
-2Lshl0PVIiTbQFDOtHheApiV/fZ30rXNlWV1zUsug/3zZFmzICc+7iDS9IxSAooV
-6JMAImjvst+uFjks5PzUD2XCGf35z2Typ1iEXkHzHTwB6+KVUu32L+QDBaJ0d31E
-no66F3iMmQeg4Lf79d7DgUOtRhbdJ++bF9sEpXPDb5Fyxt8acKj2eybPE2nXOxyt
-0e5LAqUcM/qyR5+rBL1WUsevkuRM08O0zVqg3nM7JkfmD2Eb3sLBfAQYAQgAJgIb
-DBYhBDF2732yNn8fyk8wax+bDpCa83KFBQJjxyuQBQkI0+7DAAoJEB+bDpCa83KF
-F14P/R8+Etr8EmteZq61t7Mg1rdGmddJaCzYt5C/WSeEPeTI+uCSke6gJM30N0O5
-sIp4v9rXQ3jIPJIOMkPnjggGIckhN3ohNE4WhZVGF5wkjVUoDsk+mLQLOhY8d6zc
-UvfieRN828gcDvE6Vr5E9o6+5SlOV2adC9UULijdAwd4tbYNHICS23RZC6+9pUKA
-2EE1TGor9yLv5nYAJOJgkp0fVERTcQmy2FlAS4R9SrQgAmw4V10n1ZxlFXNG/41U
-5x7B5vg9nNLZ/ZK2fTjWfSdtmiGUEIPNMcUs+upFtG1p84XQIHv909I6HnoGS8kT
-hE6wRWy1vkXIrdEl+hvpgoNSdxQCyLBLQfZxvWuU7pvmzC9NtcIkx/jFSx9Bj9XJ
-LQ2R/8zbkegrcS7K8Nr4C7EXdUBogfSzferCmAUBxeGLO/VeeAEgVQUyu+78XFuH
-SHdmT8YtC1Oe4y95iX+kkAVQNbNPxzWjXO67bnopWKnCJFOsKWLW7VefhuzB790c
-F5kXQNZGG3NxDP2H4xIytmxgikayuHKY6/ASH8T/x+poqE7S2alW+Tt7ou5ANdRk
-uH1l7qxizFjlT5nFQFJv5WfOiOa1/afVeOWdVYjpBiChn6uUKmb8XQ0ab/oX7aTq
-1niAgfxWQfRYZEt1kthCh2qnFpOZSraUYD3LFQ+GZNU5ZougwsF7BBgBCAAmAhsM
-FiEEMXbvfbI2fx/KTzBrH5sOkJrzcoUFAmHSMwsFCQb/674ACgkQH5sOkJrzcoVy
-uQ/4wTAgHaCVVvWouLSoDWnN7MRY67Zyz6OpRMweAF55bZAtVcFC3gy/jtyLnmAr
-e5Mka/b76IBv8v3vJKVSBrx/E3GXe9RL1JDOw3/V/PCOWz6wO/DKVgBEBhNUcbBS
-0FaacwDnol1U18v58Ev/Cyj9vFuRTRR9DPamJr043ktGGMMa+wyVOdugdCRoorOJ
-F5fNbWJ3dh+OnMdao0jJljFdHT1jG1BuYLNx7VOLuQ6QYJ+2vpJYoaZrcXm1jVdr
-zt0wqmCabPlm7wzx6VLBxaVvsGLxNFaLAfY2Cs4TkveGhs+GJe302av7ReGrQUXq
-HhEiYukSVqbzZnUkD2KybtwQBlp2E7p4lvcYR4UmJwkqx91n2Unq5lOl4gJirWQe
-V8OB8o7iYCo1A481Qzr6U0FXJ6ZAV5DncNHSGQ5OtXKvNaf0em5sRBrCJDBzacUX
-wNTzcI8qYIqCxdgsquZAdyFXPCnaVuDt12lUojH715ZL6UCZgj7cbqfCKlJoC/iC
-06iyWhzoaW2+m7IpEw3pFAvQyFefMUOR7hyNKmk6w8mbBvAiun/jsMSm1yYeUAxW
-qzOWKziB2jmgUiAkLQoAgR/yad/PCzpvK0w6khwEMafWY0Gn8iwWhh5R0dq7587X
-1dSx0vvT773p+idxY+K+Yc7SV7ilKzmfwbC8PHOw5ziNFMLBfAQYAQgAJhYhBDF2
-732yNn8fyk8wax+bDpCa83KFBQJc1HBNAhsMBQkB4TOAAAoJEB+bDpCa83KFDGMP
-/j/LjzcdTfiHWHc6E7EUM3qPWf8obSL7Ft4l77x0vUGf2G3pQcngTI1SIMTTLAKk
-Xhd6qPqCVPmM6kHK6IzwFcnMRFoMyoH/bVnZkUs0NyU3DPg3OUc1Iunvcg27nHdZ
-PLFRv8ey/qSyNiIEJu3hzyBIUO0ZDdOtUwkqnznrri+IpToD7gWoYM0CC/Aero/O
-aC20c6dU1s4zwmAjqfzb0Nqiv3CDrrvF0p3g6fn7BAyHxnYbS7ZXS8nPQEY0qp+y
-C0CR3jceXCwv9C1PhQiSfqiPBTL7CglOz02WSAxY7GInh3VitM2rruKcacpLVfji
-ZmFH4SUCys/7c1Sn+pJTfiqO/2sV4vutxfu3Q0xDYmcf7DK9BN7bZ01m3szTX/+5
-Ief0kpY+e5ZrfcRHUOAzA/dXeW8sErf+YvCU9Hyi/e5iWvbhaMg9HwMA37cEfhBm
-VwGBOS6nuFHn7TFoZrCNnFWEpfUJY++TThhNaVKlz5n3PXERFCJlfZtXf097cJJR
-JniBoA2jdfQqSJAgXArbZPxRW0ohIfgj+lnvqNwB27trdnKKpxC6k6P1k0QZ1MP3
-tDRaz/k0WrVi4Sxps78/RzA7I9nAR1ovVUx8Tw2I9ru64SyyyYuaA2M5nQs4kMzA
-3P3oeFO9t91by/d/O1lj9HtGYEn5xLzb40OyTfeDSyTp
-=gYkV
+sXidTQ/+Ns4PriL1io26cpZKu7CMplwkJn5xIyoV+tFJHbrgmZ7csdo1QgLHyI0L
+CA/vIw22XCrT09it8anQYkbz/3JHQAnMtsJOZI8j6j/iITomNVcZZL334YxD0NB+
+aFJEZivIa9OCvFU5gQCgIbzio1PP6Dt2PozKR63v/6hv0kp3SVlknWLRTkK3Cwbt
+YhPmfkrN9HJVzyz+6JbUmTPk28AannszumPSA7fIMeIKmpgkTeDsozN6tlcqBDzR
+ks0kQlHMzhGk5upCfgWf7aiqTF+clVSMyTiXLW+3xjN+AQHn4gHRB8buAoWaPV+J
+5vZF8xU6VL6y3fewfARkYLmZNy5Sp30ataBu5LPVPP61PMb0gskhAD/tYEC1oTuC
+SX7eWSYEA/GGjUkANXbRy1WL8YuDuckK5U9FHupkb3tZFtfuw/Zpz5rf+RzjBAPr
+AtCDkUCQD5Wg2F/KfVPrRxKCJ1va+21C6ngwmQKghDiIcjMUps9vnfixBOhTU2hg
++woYnKnOOkg+0TD7zJPO0gV9s39W5B/2fdDfIprnq6dk5tJXcqIVUtnxlqUam1cM
+BKkgqSXQxLwvMUOjFBH6MxXmJwW3OVBZN+GYXFsVnDtCcHNqbEGEiQMIh+hhBbrS
+tc+PiRCaudkiZ11ltx11njEdV7VbbGxepus3mTQuxVxU1GFdS3jCw7IEGAEIACYC
+GwIWIQQxdu99sjZ/H8pPMGsfmw6QmvNyhQUCY8crlAUJCNPswwJACRAfmw6QmvNy
+hcF0IAQZAQgAHRYhBMuN5wqQz79sO/XMVpYmKs/7067GBQJc1HJNAAoJEJYmKs/7
+067G2s8QAIP/MH/Xzbuhz7uO+6FFNS7kk00zQImC1Y0yYyeyy2UWsSD4HUdE05Co
+JCMSwHXpTNzDs/aackTsKivEINYPvTwbEGPMPqv9MVD78T46iwSuA+Qg32CDLAju
+bby8Q55DXYS/q40CZaLzg7OQT8IH03ewxzrtpdLrnXnYN1ktwBFokv5ZsxB5BhEJ
+fbEHEqdoNk2STQv5p/Ikrc5C/hefHSKXV4cJRYoPkrdnr0ZHyevB5iGuR9zMxmkL
+D+NyIqqVqxjWQNjLoEdY+xKnezHiIZvGb6nH55DosZY5/IHgLUJsLFcYNfLFSVdA
+zF0py+A5nY0PJarlZptOi22tSsSXX5EKWHLTZUUcs1uBrmydVFMsT3ZC+8pHxLyK
+Gn+f+89jdUCzidQ+545OYnNTzxTjGytG9Y/KEVDYUWAm9OSSLJjDpt4RDVF6yte8
+cHJfdC9Ypazet2Z2GMtGa6g6dv7NhDF+JCEhnMraPaLHbWnABP47AJZ1cRvYfE2l
+yHBzfKWyWJ56BWH9Sf7p6DwehqIAo0erPAcnBDcbrfrHaSnFLvW9UpIGAGvBMXZq
+AoZNNsXqq7OMHRt9rWZXOGkeGAINnwlQIT3ffKMounUS9xzLM0kZdf7BS7K5+5gc
+jJPh1b1yvBYApioBAXbnaTmqvdwr03FLnoGi5/0671t0iUGSqbF4gjYP/jOOTWQ1
+6ImKxAbQGLoStrlU3ksDEvgE3Ot1BHkH+jLZWIs7f/OA9SM9j8aYqcHT+UCgiAiq
+DoU1axEf2AefpQi+5t5Kllg42vHT3VILkWIby6LldYZzRcPKDdis2YX78WlRCpcN
+6poZTcAx5z6Msvj0M3UdcgjshBgqtaa2pfwpGp+pmUh3tlz4Kqp+wTrooUsytuBL
+E/pvSE4cgt83j1Tu0bUwQQpfaFrfqGWbe9RGbSRNqLxPqKyackSumGvzmOjXUrzj
+b25/RAbWZzC1xVQT0/E7HbQSNtdi8aE9NxgsEVfH9iXHuxOOkKJMqHkrmFoyrFSh
+YsjD8OmHCes3WsEVFgtu6ecUPvAcrDn3STC5Q71Gkoe7slbT8vBEJoENARCo1bKl
+8zzgIiNiuKQ4q7D/coCTBEbw45QFVJV+YoDqWUbpibbHC4Ox9heHo1qkjnXHjEsT
+NyUp9VN2r4aKVXrWdmb2ec+LF0Lt8SZScoLcr/OP4bXncHcokqTh4304A7myJXkS
+ZqiVjssdkkts4KPKkMAo2Wx30ZjaBDTo4oRm2iK3rXFI6LS4ClUBG10JlwPo98Dk
+k9yzQxCvIMIGAS/PTCm1osxy7kZWDpiJb4wGBq0BZiCy23k/7TkeqkC/51z7U4mG
+ZARfQI95wwwHxLURgJzJsuh8G7gkTcHL01kkwsOyBBgBCAAmAhsCFiEEMXbvfbI2
+fx/KTzBrH5sOkJrzcoUFAmHSMy4FCQb/6eECQAkQH5sOkJrzcoXBdCAEGQEIAB0W
+IQTLjecKkM+/bDv1zFaWJirP+9OuxgUCXNRyTQAKCRCWJirP+9OuxtrPEACD/zB/
+1827oc+7jvuhRTUu5JNNM0CJgtWNMmMnsstlFrEg+B1HRNOQqCQjEsB16Uzcw7P2
+mnJE7CorxCDWD708GxBjzD6r/TFQ+/E+OosErgPkIN9ggywI7m28vEOeQ12Ev6uN
+AmWi84OzkE/CB9N3sMc67aXS65152DdZLcARaJL+WbMQeQYRCX2xBxKnaDZNkk0L
++afyJK3OQv4Xnx0il1eHCUWKD5K3Z69GR8nrweYhrkfczMZpCw/jciKqlasY1kDY
+y6BHWPsSp3sx4iGbxm+px+eQ6LGWOfyB4C1CbCxXGDXyxUlXQMxdKcvgOZ2NDyWq
+5WabTottrUrEl1+RClhy02VFHLNbga5snVRTLE92QvvKR8S8ihp/n/vPY3VAs4nU
+PueOTmJzU88U4xsrRvWPyhFQ2FFgJvTkkiyYw6beEQ1ResrXvHByX3QvWKWs3rdm
+dhjLRmuoOnb+zYQxfiQhIZzK2j2ix21pwAT+OwCWdXEb2HxNpchwc3ylslieegVh
+/Un+6eg8HoaiAKNHqzwHJwQ3G636x2kpxS71vVKSBgBrwTF2agKGTTbF6quzjB0b
+fa1mVzhpHhgCDZ8JUCE933yjKLp1EvccyzNJGXX+wUuyufuYHIyT4dW9crwWAKYq
+AQF252k5qr3cK9NxS56Bouf9Ou9bdIlBkqmxeLF1D/9hJ0VQU3BjaBVJTKYsfZsv
+SDLxgqGHFdz5TzyLnGapshR9ZMvoyjUZUylupYpvnbKx6ypxZEP7ML+nWZ45KYdL
+SsaqXHN8mCInlD6MvcqXXccuUNffcgQXHhlP/lE2DFqlYDjhC13m5UyAqFfhk73k
+PQ5MaUX9lPZCfFYW+YVAOJlAtlns/fbnJY8EJe9msj+0aAyENdhq61+XxUIvtZp8
+KiT7vMtgFHgkguw888n4GQvYSQ/DvCnh9N1wnn6En8QDuBx/RK9VDfU0hUs7CH7S
+vxuHFvF6Koe0v+bS2kZO/3QhnLZhDNOkm2nMImRYGD+8QJGIIOebRl/FC8VUCOCq
+Y1z90S2sOWt0VFJvUoO/m0IFNYwdKsati7l6TzFpmqYl+rlsfDya1idtVzZx5nLM
+8Erck1gmDGh9WXHasuW1nvaJJSClQbrPJNiLtEZN+IZj4Ggd0xv8Mjq2lJzWXsHH
+MRXPZpbOaNb+PJa5uGREVvXY8FCI/3glOWd327m3ZN9keRPTbzJXMXBPuUhyhSXd
+B9nFWT54SORvdD/3kBQJJhoaRraUkPQJh/az/ivxgdSY+BjNrtyA8DIHbCYUxlHy
+YdVFuqYQceseVjDf568pRb+1O7KYIBX0rZStwJisus0d8ZboJppinOuVwk5fyhu8
+LpxuVbrRPQQsZb4lSnhOXsLDsgQYAQgAJgIbAhYhBDF2732yNn8fyk8wax+bDpCa
+83KFBQJgizOSBQkFl/TBAkAJEB+bDpCa83KFwXQgBBkBCAAdFiEEy43nCpDPv2w7
+9cxWliYqz/vTrsYFAlzUck0ACgkQliYqz/vTrsbazxAAg/8wf9fNu6HPu477oUU1
+LuSTTTNAiYLVjTJjJ7LLZRaxIPgdR0TTkKgkIxLAdelM3MOz9ppyROwqK8Qg1g+9
+PBsQY8w+q/0xUPvxPjqLBK4D5CDfYIMsCO5tvLxDnkNdhL+rjQJlovODs5BPwgfT
+d7DHOu2l0uudedg3WS3AEWiS/lmzEHkGEQl9sQcSp2g2TZJNC/mn8iStzkL+F58d
+IpdXhwlFig+St2evRkfJ68HmIa5H3MzGaQsP43IiqpWrGNZA2MugR1j7Eqd7MeIh
+m8ZvqcfnkOixljn8geAtQmwsVxg18sVJV0DMXSnL4DmdjQ8lquVmm06Lba1KxJdf
+kQpYctNlRRyzW4GubJ1UUyxPdkL7ykfEvIoaf5/7z2N1QLOJ1D7njk5ic1PPFOMb
+K0b1j8oRUNhRYCb05JIsmMOm3hENUXrK17xwcl90L1ilrN63ZnYYy0ZrqDp2/s2E
+MX4kISGcyto9osdtacAE/jsAlnVxG9h8TaXIcHN8pbJYnnoFYf1J/unoPB6GogCj
+R6s8BycENxut+sdpKcUu9b1SkgYAa8ExdmoChk02xeqrs4wdG32tZlc4aR4YAg2f
+CVAhPd98oyi6dRL3HMszSRl1/sFLsrn7mByMk+HVvXK8FgCmKgEBdudpOaq93CvT
+cUuegaLn/TrvW3SJQZKpsXjLuA/9E35tq9kEQLfVk/XIPaNcK1cYthOICf/LfZVc
+NvlGxIfMGfuEbQ+1eWcdVa/UW1Kff3VUOZaAjyRbpdrVbEUCyoFnnMEs/GbCG5+g
+WGpgwD6jt7tESCCpQIssp3b8vf69SWNH8jKY4LXPkeSaxuhFWlUjaJXnvCMYWeHT
+PTkeBKHVMoAKYWKr7t6jgLDGoNO+B0l4vVGq2K8M0obd5Wn8HV1IgYu9yP5CX8Kz
+YkSyg9Vc2djQ4k9aKfCthVwwKJ+OH1MJFpEKYihmUweaGo9+32sHdT4ifyn6zS+K
+8HHZGlK6DeBlqrWkx7GPwHZWtq6v2DxcobZieLoLmyDZlJFJHjbKZ9Bg/OqUaqCy
+uFzHoVC0UTRIgvblButt1agaMPz/7+VdXZUKmXjZjopW6R7ScxK/q4uvKykY+r0e
+DTihLp6Nyb7m96Xv8HvlBe9Vzxm//PtosZUssQatJHMecKePEhkY0i/bqm4CAur7
+ESfWatgZhZC0MIG0jTCB6O7ueBVTKX80eXeyErYt+WraiH6wuBW8GCLuuMnAiaVk
+oknx75Oyqirr3Una2xoGGMcER2+QTWTxD/GuiPIOK6z1ktglCjIHhK+bedsSrv6p
+nJtcdc42btv57ZmslyjiSIOwnCHQNfODjc9Ke4/FtVnsRmIhn/NXI8OdL5oHvI4H
+u/sWWdHCw7IEGAEIACYWIQQxdu99sjZ/H8pPMGsfmw6QmvNyhQUCXNRyTQIbAgUJ
+AeEzgAJACRAfmw6QmvNyhcF0IAQZAQgAHRYhBMuN5wqQz79sO/XMVpYmKs/7067G
+BQJc1HJNAAoJEJYmKs/7067G2s8QAIP/MH/Xzbuhz7uO+6FFNS7kk00zQImC1Y0y
+Yyeyy2UWsSD4HUdE05CoJCMSwHXpTNzDs/aackTsKivEINYPvTwbEGPMPqv9MVD7
+8T46iwSuA+Qg32CDLAjubby8Q55DXYS/q40CZaLzg7OQT8IH03ewxzrtpdLrnXnY
+N1ktwBFokv5ZsxB5BhEJfbEHEqdoNk2STQv5p/Ikrc5C/hefHSKXV4cJRYoPkrdn
+r0ZHyevB5iGuR9zMxmkLD+NyIqqVqxjWQNjLoEdY+xKnezHiIZvGb6nH55DosZY5
+/IHgLUJsLFcYNfLFSVdAzF0py+A5nY0PJarlZptOi22tSsSXX5EKWHLTZUUcs1uB
+rmydVFMsT3ZC+8pHxLyKGn+f+89jdUCzidQ+545OYnNTzxTjGytG9Y/KEVDYUWAm
+9OSSLJjDpt4RDVF6yte8cHJfdC9Ypazet2Z2GMtGa6g6dv7NhDF+JCEhnMraPaLH
+bWnABP47AJZ1cRvYfE2lyHBzfKWyWJ56BWH9Sf7p6DwehqIAo0erPAcnBDcbrfrH
+aSnFLvW9UpIGAGvBMXZqAoZNNsXqq7OMHRt9rWZXOGkeGAINnwlQIT3ffKMounUS
+9xzLM0kZdf7BS7K5+5gcjJPh1b1yvBYApioBAXbnaTmqvdwr03FLnoGi5/0671t0
+iUGSqbF4pycP/j7UdSDlwqt9JNdk2y45TsCeXJtalTsmXxh3F964IJ74jlbeNNiq
+WOkJEZYEAe0Te50HfslcpKfzIfvQnNdANYv34bXCqTmLCS+2YqoBqnMkW5n2Vg2U
+0ZgPEj1+hOr31/46F4WdPedSbwgD03QzCMYIgnnnTUcz407xbKVWjLRj56vFvJAY
+LYTLtiNVlK4JNmjcG5iJII/wGZ32ofEImYEZCBu0D/ZAFwtnyj/P3DRTf0jCx6Ly
+hpxmdxVMgWcEOxTcTbkzV7JDxV1wCosvu/dQU6L7bTMEGz6DyMEF3NdRWGn5ECxe
+V2jyL+q10Pj/3RRdPsUFeYiYa3M5WbNrbe/sPKrs/9m54QCqGvu+1b3tRDv5iRcN
+riseLgesYkKWt3GLETIJM+sP75K/DEC53eEBQpkX2E5vR8P0DtyfrcQRFWBokYCL
+un06qpuqeLwk6yxAXpN4JQU/5fedHj6LLwRfixx83otQHr+aeGhFVncY0TM8BmL0
+8Z7xGpYmpQy2sFVQ9XFzACB+KM+aYwhchTJmTnKxNB5Pd4lV3dHfLE5F9odIPmS4
+eddbCyn/JiW2wsj+EG1Vrw3tdl3Nn0kNPx8haPnnXkOx+6dO2Sys5ulF2KriXECh
+iqJugj4vUEIfC9pPL2CFeGGmCfKbQMmTZua5xFqKv3pqlPN1iizN3uIszsFNBFzU
+cE0BEADoTMBwOhcQ43cSw5z2GWfx0Qgd4/hZ+bPnP6oityeeBuMmD/dfO9RYw3he
+6Q7JnEEYAs9JOhnmTsxTpKQkxoOmjD5DogaPd9TqnJM4ZQApy0gs0eW9myC02lBp
+3+LeNQ+vrQhStnXyQXUNMmHwd5WvJIPWu77Z5WjPVDtE6/qPTWDlsrFAGuX8UvY0
+hdgs4HmepPVmtG4/dLvDfXH7BqzAexek1TxuE3+xmwr4DMGgX0kCu+IV1VndMI01
+tFRiKcUXwGLnScgf9nWuazrDFoeYRHunUocBENbSR547gSCSNVGUslbRajI/DcrH
+7Q+/la3ejdxaebplM3OrVYmoPrurPEPE8QlzGksKFbQVdRx26iogs8HeFzcB37/4
+dFa4DNEl62FZh8FJC1DXTpQClkWexMWkzMdKDWisI69xYVe1ubrX+pEPZTH0vIFd
+HSTNZ2adGZIIv6I7iHnKgUFycE6dWEKNUJiRzNLCk7uqTjpU8Ys2k+AJNHrUH/Ui
+4p5bTHmBHQqBWr1pFlyCD+GDPeH7g+RJHNN29oogjClTjUq+HSmgDxRLVd7hSKEv
+1cBuUEPJhIWWR/cpjnlK5vmL5YxUn6dUw6fXRomDSgsedmY+8AoarMM1RFRU6hhG
+arGLcrrg0DNCYV+kxk++c5rKoiYQRpep4oBq3WkQp1r1r9J9HQARAQABwsF8BBgB
+CAAmAhsMFiEEMXbvfbI2fx/KTzBrH5sOkJrzcoUFAmlRhdwFCQ5eSQ8ACgkQH5sO
+kJrzcoWEBw/9G0iQrixwk5s4GdVwle8PxAYzsh1IpWrYGR5GX2aMJ9n8kLgypeR3
+UqVK3rZDPmOZAPi3IWcyD72XWzjYKBhAx8YQU41IAlrUSWMEZt8VkaZwmG4HHzJw
+hMgAax3Pjkw8vPZYJcT7s197McVbKILGZH5fXt4kFVZItCEGblKMbOVPWdPWtxRi
+lvysf/mwRYmiq4pDiC8J4aWLHeoPjZggFRkSR6IK0F0i3dj9TjuT3ik2O3I/jA8z
+z9yRQ2Mm2hWJ7QfZ536cPtZsEpRojScbKixlsG3l0PV/yWbBU9ZKA9XJdO0XV82P
+AC/3B312p7scbUF/UAwEQkO9snq9OuLxBQtaJH5wueF5Dx+SmrGdZw2kkEnUdoKL
+5xk6xsgIh4lbQeEO4EiCUXJ9fQ6Yx/ycqXoDRg958puRt9nlkHvwIOsrtn1qqPDX
+XiEAedzVsZX9CS4edNYinDi/qYzAC9LiXf9M3b19dOBq4u+oXg0y4kxh/H40wgoF
+VjsjM62Z+2dMJVDWItASASMNlms2ScDXYqe91vyhg0kyUu+yLZxyeRldP4IxqB7H
+eONCwYhzREyFYXsCSemJux5rWgS/r/PZhcDimrAmTTqR4UgajVTEFaOWehdHMYFY
+BX1MenT04F+6r7yr0sb0/zu9Nr+twzEQEYb0D5+K0dyECsA2d4PCaA3CwXwEGAEI
+ACYCGwwWIQQxdu99sjZ/H8pPMGsfmw6QmvNyhQUCZZgiCgUJDIYYvQAKCRAfmw6Q
+mvNyheNyD/9uERCjRwJzBaF7eOl64Rqt27mYLgJKU9eA+4xit5ZEATPAlTuIbbqp
+0edYQLo2peMOgAADWmSYujOTUK7+e0/hHo7Mjr2RwbTnKOtKrG7tj4emqSQ4la0L
+GkJ4J1RoRHPLQw+VDOS7XHkidXwU4RJzz20nZGXVfCHmbvNOzoqbhUC36Sh6dONF
+cyUNLk30crABh6vGVqKRL9KM2A3CH3RU9reLb4khKgTUlDWt0g9itU5V6rih2dyE
+kApDt0OOP3ksywTCqJNBSnVYopVn1IlkEGbCKEfeANJteDQpu3P5RkCu8pfvkYnO
+dik+FbmfmNCtfe67lmgT0uzdfMRzR9Pz+24Tkhruzxlt30m14n+8QHwfZqCAkedB
+iyG3vf4guOunrtn6RgeeXZHfKBgy/xhsgg/RJpDUkoGeGDSrdn0elRbSXc57Wayt
+onIa2Lshl0PVIiTbQFDOtHheApiV/fZ30rXNlWV1zUsug/3zZFmzICc+7iDS9IxS
+AooV6JMAImjvst+uFjks5PzUD2XCGf35z2Typ1iEXkHzHTwB6+KVUu32L+QDBaJ0
+d31Eno66F3iMmQeg4Lf79d7DgUOtRhbdJ++bF9sEpXPDb5Fyxt8acKj2eybPE2nX
+Oxyt0e5LAqUcM/qyR5+rBL1WUsevkuRM08O0zVqg3nM7JkfmD2Eb3sLBfAQYAQgA
+JgIbDBYhBDF2732yNn8fyk8wax+bDpCa83KFBQJjxyuQBQkI0+7DAAoJEB+bDpCa
+83KFF14P/R8+Etr8EmteZq61t7Mg1rdGmddJaCzYt5C/WSeEPeTI+uCSke6gJM30
+N0O5sIp4v9rXQ3jIPJIOMkPnjggGIckhN3ohNE4WhZVGF5wkjVUoDsk+mLQLOhY8
+d6zcUvfieRN828gcDvE6Vr5E9o6+5SlOV2adC9UULijdAwd4tbYNHICS23RZC6+9
+pUKA2EE1TGor9yLv5nYAJOJgkp0fVERTcQmy2FlAS4R9SrQgAmw4V10n1ZxlFXNG
+/41U5x7B5vg9nNLZ/ZK2fTjWfSdtmiGUEIPNMcUs+upFtG1p84XQIHv909I6HnoG
+S8kThE6wRWy1vkXIrdEl+hvpgoNSdxQCyLBLQfZxvWuU7pvmzC9NtcIkx/jFSx9B
+j9XJLQ2R/8zbkegrcS7K8Nr4C7EXdUBogfSzferCmAUBxeGLO/VeeAEgVQUyu+78
+XFuHSHdmT8YtC1Oe4y95iX+kkAVQNbNPxzWjXO67bnopWKnCJFOsKWLW7VefhuzB
+790cF5kXQNZGG3NxDP2H4xIytmxgikayuHKY6/ASH8T/x+poqE7S2alW+Tt7ou5A
+NdRkuH1l7qxizFjlT5nFQFJv5WfOiOa1/afVeOWdVYjpBiChn6uUKmb8XQ0ab/oX
+7aTq1niAgfxWQfRYZEt1kthCh2qnFpOZSraUYD3LFQ+GZNU5ZougwsF7BBgBCAAm
+AhsMFiEEMXbvfbI2fx/KTzBrH5sOkJrzcoUFAmHSMwsFCQb/674ACgkQH5sOkJrz
+coVyuQ/4wTAgHaCVVvWouLSoDWnN7MRY67Zyz6OpRMweAF55bZAtVcFC3gy/jtyL
+nmAre5Mka/b76IBv8v3vJKVSBrx/E3GXe9RL1JDOw3/V/PCOWz6wO/DKVgBEBhNU
+cbBS0FaacwDnol1U18v58Ev/Cyj9vFuRTRR9DPamJr043ktGGMMa+wyVOdugdCRo
+orOJF5fNbWJ3dh+OnMdao0jJljFdHT1jG1BuYLNx7VOLuQ6QYJ+2vpJYoaZrcXm1
+jVdrzt0wqmCabPlm7wzx6VLBxaVvsGLxNFaLAfY2Cs4TkveGhs+GJe302av7ReGr
+QUXqHhEiYukSVqbzZnUkD2KybtwQBlp2E7p4lvcYR4UmJwkqx91n2Unq5lOl4gJi
+rWQeV8OB8o7iYCo1A481Qzr6U0FXJ6ZAV5DncNHSGQ5OtXKvNaf0em5sRBrCJDBz
+acUXwNTzcI8qYIqCxdgsquZAdyFXPCnaVuDt12lUojH715ZL6UCZgj7cbqfCKlJo
+C/iC06iyWhzoaW2+m7IpEw3pFAvQyFefMUOR7hyNKmk6w8mbBvAiun/jsMSm1yYe
+UAxWqzOWKziB2jmgUiAkLQoAgR/yad/PCzpvK0w6khwEMafWY0Gn8iwWhh5R0dq7
+587X1dSx0vvT773p+idxY+K+Yc7SV7ilKzmfwbC8PHOw5ziNFMLBfAQYAQgAJgIb
+DBYhBDF2732yNn8fyk8wax+bDpCa83KFBQJgizOOBQkFl/bBAAoJEB+bDpCa83KF
+TV8P/2YkaC5MU7gnWckBHb+fgNFXrYv8iNo5GF4+qKLDkU4E2vn/36CCPzJXsiXg
+xT2hZMzUNFxgx8c6+wDbJLmGsc/KJ9YelxOqS52d5w8FPUmdW8yK/Nf6zjHWiXgT
+I5BEHIGhgIWwRhpGxw2sMhMyJ3jNY/LuuRU8OvVEH5wlviVDN9HHpcbxsm4XwzKY
+R31LkY3F4c2KB2cU/e/dDCGKpJptYem6X6TlV6VJGo6V9U80KgIAEh93mcQTCTOm
+d2yZFGPHdCJ2SHsPCn+X89AL9AzpEz6b/culcn5dqHcOqRkergRiGarhCYuWPMyh
+P0RRfFaGIHqvxd+D2RccsTDt953mcH2VejVSCcSJaIv0Wlz/tl+beGaQ72nUI7hI
+kI7rMdg2kvjrdnpRKc3+0sVwkUAkk2kKFdEc8XiDZgTLiCzwgM2bOPI4wsNye32/
+oM23bdJXF2Lk+vGU+8tMDT22M5CVHaNMKx8Rb3oXsATdcqiu92bgaRTuNYV4jO/d
+GWllkhnEaB8e8nSAQDVQYqF1X2pmHjrLmLYzAYgunsNmCvV7D16Si2aiqQ7oaHpL
+EjV3dz4h5Lbr5U2hxWNAWNC9+gnQK1Hb0BUI/mNhm0V5UiHCkuahoh19MqCj9BU6
+3OV0rh0LEU1KGIJiPeAO/lnIDPHhBhaoJPuSYnvR8PMyCeKlwsF8BBgBCAAmFiEE
+MXbvfbI2fx/KTzBrH5sOkJrzcoUFAlzUcE0CGwwFCQHhM4AACgkQH5sOkJrzcoUM
+Yw/+P8uPNx1N+IdYdzoTsRQzeo9Z/yhtIvsW3iXvvHS9QZ/YbelByeBMjVIgxNMs
+AqReF3qo+oJU+YzqQcrojPAVycxEWgzKgf9tWdmRSzQ3JTcM+Dc5RzUi6e9yDbuc
+d1k8sVG/x7L+pLI2IgQm7eHPIEhQ7RkN061TCSqfOeuuL4ilOgPuBahgzQIL8B6u
+j85oLbRzp1TWzjPCYCOp/NvQ2qK/cIOuu8XSneDp+fsEDIfGdhtLtldLyc9ARjSq
+n7ILQJHeNx5cLC/0LU+FCJJ+qI8FMvsKCU7PTZZIDFjsYieHdWK0zauu4pxpyktV
++OJmYUfhJQLKz/tzVKf6klN+Ko7/axXi+63F+7dDTENiZx/sMr0E3ttnTWbezNNf
+/7kh5/SSlj57lmt9xEdQ4DMD91d5bywSt/5i8JT0fKL97mJa9uFoyD0fAwDftwR+
+EGZXAYE5Lqe4UeftMWhmsI2cVYSl9Qlj75NOGE1pUqXPmfc9cREUImV9m1d/T3tw
+klEmeIGgDaN19CpIkCBcCttk/FFbSiEh+CP6We+o3AHbu2t2coqnELqTo/WTRBnU
+w/e0NFrP+TRatWLhLGmzvz9HMDsj2cBHWi9VTHxPDYj2u7rhLLLJi5oDYzmdCziQ
+zMDc/eh4U7233VvL9387WWP0e0ZgSfnEvNvjQ7JN94NLJOk=
+=sTyE
 -----END PGP PUBLIC KEY BLOCK-----
diff --git a/expat.spec b/expat.spec
index 8c005d0..8c81bd3 100644
--- a/expat.spec
+++ b/expat.spec
@@ -13,6 +13,7 @@ License: MIT
 BuildRequires: autoconf, libtool, xmlto, gcc-c++
 BuildRequires: make
 BuildRequires: gnupg2
+BuildRequires: openpgpverify
 
 %description
 This is expat, the C library for parsing XML, written by James Clark. Expat
@@ -39,7 +40,7 @@ The expat-static package contains the static version of the expat library.
 Install it if you need to link statically with expat.
 
 %prep
-%{gpgverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
+%{openpgpverify} --keyring='%{SOURCE2}' --signature='%{SOURCE1}' --data='%{SOURCE0}'
 %autosetup
 sed -i 's/install-data-hook/do-nothing-please/' lib/Makefile.am
 ./buildconf.sh
-- 
2.56.0

